services/ossf-scorecard/ossf-scorecard.tester.js

```diff
@@ -0,0 +1,17 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('score valid')
+  .get('/github.com/rohankh532/org-workflow-add.json')
+  .expectBadge({
+    label: 'score',
+    message: Joi.number().min(0),
+    color: Joi.string().allow(
+      'red',
+      'yellow',
+      'yellowgreen',
+      'green',
+      'brightgreen'
+    ),
+  })
```

Comment:
Please add one more case that hits the 404 path, presumably by specifying some non-existent repo

Reply:
Added a test for this